.travis.yml

```diff
@@ -26,15 +32,51 @@ matrix:
 
         - os: linux
           sudo: required
+          python: 3.3
+          env:
+            - SCAPY_SUDO=sudo
+
+        - os: linux
+          sudo: required
+          python: 3.6
+          env:
+            - SCAPY_SUDO=sudo
+
+        - os: linux
+          sudo: required
           python: 2.7
           env:
             - SCAPY_SUDO=sudo SCAPY_COVERAGE=yes
 
         - os: linux
           sudo: required
+          python: 3.3
+          env:
+            - SCAPY_SUDO=sudo SCAPY_COVERAGE=yes
```

Comment:
I think it's duplicated to have both
```
- os: linux
      sudo: required
      python: 3.3
      env:
        - SCAPY_SUDO=sudo
```
and
```
- os: linux
      sudo: required
      python: 3.3
      env:
        - SCAPY_SUDO=sudo  SCAPY_COVERAGE=yes
```
It really changes nothing and overloads Travis :/ Same for Python 2.7

Reply:
That's how it was configured before, no idea why.

Reply:
Also, maybe when a test fails, it's good to know if both versions fail (maybe we just have to run the test again, or maybe the failure has be caused by coverage itself, etc.). That might be the reason why it was configured to run with and without coverage.

Reply:
That's likely a mistake as I don't remember why I did it this way. Only the coverage test should be used.